docs/data/material/customization/density/DensityTool.js

```diff
@@ -9,14 +8,20 @@ import FormControlLabel from '@mui/material/FormControlLabel';
 import Switch from '@mui/material/Switch';
 import IncreaseIcon from '@mui/icons-material/AddCircleOutline';
 import DecreaseIcon from '@mui/icons-material/RemoveCircleOutline';
-import { DispatchContext } from 'docs/src/modules/components/ThemeContext';
+import {
+  DispatchContext,
+  ThemeOptionsContext,
+} from 'docs/src/modules/components/ThemeContext';
 import { useTranslate } from '@mui/docs/i18n';
+import { setDocsSpacing, resetDocsSpacing } from 'docs/src/BrandingCssVarsProvider';
```

Comment:
Similar to the color playground, spacing is available through a CSS variable `--mui-spacing`.